jsapp/js/components/reports.es6

```diff
@@ -759,24 +759,26 @@ class Reports extends React.Component {
     stores.allAssets.whenLoaded(uid, (asset)=>{
       let rowsByKuid = {};
       let rowsByIdentifier = {};
-      let names = [],
-          groupBy = [],
+      let groupBy = [],
           reportStyles = asset.report_styles,
           reportCustom = asset.report_custom;
 
-      if (!this.state.currentCustomReport && reportStyles.default.groupDataBy !== undefined)
-        groupBy = reportStyles.default.groupDataBy;
-
-      if (this.state.currentCustomReport && this.state.currentCustomReport.reportStyle.groupDataBy)
+      if (this.state.currentCustomReport && this.state.currentCustomReport.reportStyle) {
         groupBy = this.state.currentCustomReport.reportStyle.groupDataBy;
```

Comment:
If `this.state.currentCustomReport.reportStyle.groupDataBy` is `undefined`, should `reportStyles.default.groupDataBy` be used?

Reply:
You're right, I've added one more check to it :)

Reply:
Thank you!